lib/bank_account/account.rb

```diff
@@ -0,0 +1,47 @@
+require 'pry'
+
+module BankAccount
+
+  class Account
+    # Makes id and initial balance viewable
+    attr_reader :id, :initial_balance
+    # Makes current balance changable
+    attr_accessor :current_balance
+
+    def initialize(id, initial_balance)
+      @id = id
+      @initial_balance = initial_balance
```

Comment:
Do you need the `@initial_balance` isntance variable for use later or do you only need one instance variable to keep track of the balance?